Src/PerformanceCollector/Perf.Shared.Tests/QuickPulse/QuickPulseTelemetryModuleTests.cs

```diff
@@ -50,6 +48,25 @@ public void QuickPulseTelemetryModuleIsInitializedBySdk()
             qp.Dispose();
         }
 
+        [TestMethod]
+        public void QuickPulseTelemetryModuleServerIdDefaultsToMachineName()
+        {
+            using (var qp = new QuickPulseTelemetryModule())
+            {
+                Assert.AreEqual(Environment.MachineName, qp.ServerId);
+            }
+        }
+
+        [TestMethod]
+        public void QuickPulseTelemetryModuleServerIdCanBeChanged()
+        {
+            using (var qp = new QuickPulseTelemetryModule())
+            {
+                qp.ServerId = "my-server-name";
+                Assert.AreEqual("my-server-name", qp.ServerId);
```

Comment:
As much as I enjoy testing the C# compiler, maybe call Initialize() before asserting just to make it more fun :)